app/invocation/invocation_action_card.tsx

```diff
@@ -0,0 +1,64 @@
+import pako from "pako";
+import React from "react";
+import format from "../format/format";
+import InvocationModel from "./invocation_model";
+import rpcService from "../service/rpc_service";
+
+interface Props {
+  model: InvocationModel;
+  inProgress: boolean;
+  hash: string;
+  size: Long;
+}
+
+interface State {
+  loading: boolean;
+  contents: string;
+}
+
+export default class ActionCardComponent extends React.Component {
```

Comment:
nit: use `React.Component<Props, State>` instead, and then remove the `props: Props` part below.